controller/array_action/array_mediator_ds8k.py

```diff
@@ -79,10 +80,13 @@ def is_snapshot(api_volume):
     return False
 
 
-@decorator
-def convert_scsi_id_to_array_id(mediator_method, self, *args):
-    args = try_convert_first_arg(scsi_id_to_volume_id, args)
-    return mediator_method(self, *args)
+def convert_scsi_ids_to_array_ids(args_amount=1):
+    @decorator
+    def convert_scsi_id_to_array_id(mediator_method, self, *args):
```

Comment:
consider renaming to something like `try_convert_first_args`

Reply:
done